src/app/Http/Controllers/BusinessTripController.php

```diff
@@ -77,4 +87,191 @@ public function confirm() {
+                ];
+                break;
+
+            case DocumentType::DOMESTIC_REPORT:
+                $name = $trip->user->academic_degrees
+                    ? $trip->user->academic_degrees . ' ' . $trip->user->first_name . ' ' . $trip->user->last_name
+                    : $trip->user->first_name . ' ' . $trip->user->last_name;
+                $mealsReimbursementText = $trip->meals_reimbursement == 1
+                    ? 'mám záujem o preplatenie'
+                    : 'nemám záujem o preplatenie';
+                $data = [
+                    'name' => $name,
+                    'department' => $trip->user->department,
+                    'date_start' => $trip->datetime_start->format('d-m-Y'),
+                    'date_end' => $trip->datetime_end->format('d-m-Y'),
+                    'spp_symbol' => $trip->spp_symbols->spp_symbol,
+                    'time_start' => $trip->datetime_start->format('H:i'),
+                    'time_end' => $trip->datetime_end->format('H:i'),
+                    'transport' => $trip->transport->name,
+                    'travelling_expense' => $trip->travellingExpense ? $trip->travellingExpense->amount_eur : null,
+                    'accommodation_expense' => $trip->accommodationExpense ? $trip->accommodationExpense->amount_eur : null,
+                    'other_expenses' => $trip->otherExpense ? $trip->otherExpense->amount_eur : null,
+                    'allowance' => $trip->allowanceExpense ? $trip->allowanceExpense->amount_eur : null,
+                    'conclusion' => $trip->conclusion,
+                    'iban' => $trip->iban,
+                    'address' => $trip->user->address,
+                    'meals_reimbursement_DG42' => $mealsReimbursementText,
+                ];
+                break;
+
+            case DocumentType::FOREIGN_REPORT:
+                $mealsReimbursementText = $trip->meals_reimbursement == 1
+                    ? 'mam zaujem o preplatenie'
+                    : 'nemam zaujem o preplatenie';
+                $data = [
+                    'name' => $trip->user->first_name . ' ' . $trip->user->last_name,
+                    'department' => $trip->user->department,
+                    'country' => $trip->country->name,
+                    'datetime_end' => $trip->datetime_end->format('d-m-Y H:i'),
+                    'datetime_start' => $trip->datetime_start->format('d-m-Y H:i'),
+                    'datetime_border_crossing_start' => $trip->datetime_border_crossing_start->format('d-m-Y H:i'),
+                    'datetime_border_crossing_end' => $trip->datetime_border_crossing_end->format('d-m-Y H:i'),
+                    'place' => $trip->place,
+                    'spp_symbol' => $trip->sppSymbol->spp_symbol,
+                    'transport' => $trip->transport->name,
+                    'travelling_expense_foreign' => $trip->travellingExpense ? $trip->travellingExpense->amount_foreign : null,
```

Comment:
ja len ze v tejto casti sa mi to trosku nezda. ak existuje nejaky travellingExpense, tak sa tam zapise suma v cudzej mene z travellingExpense, lenze ta suma v cudzej mene moze byt aj null, lebo user mohol zadat napr. iba sumu v eurach. nespadne to v takom pripade?